tests/test_simulatedelayd.py

```diff
@@ -15,12 +15,13 @@
 from parameterized import parameterized
 from monai.transforms.utility.dictionary import SimulateDelayd
 from tests.utils import NumpyImageTestCase2D
+from monai.config.type_definitions import KeysCollection
 
 
 class TestSimulateDelay(NumpyImageTestCase2D):
     @parameterized.expand([(0.45,), (1,)])
     def test_value(self, delay_test_time: float):
-        resize = SimulateDelayd(keys="imgd", delay_time=delay_test_time)
+        resize = SimulateDelayd(keys=("imgd",), delay_time=delay_test_time)
```

Comment:
Why change this input to `tuple` as `MapTransform` can support string input and has `ensure_tuple` in it.
You want to enforce the input to `tuple`?

Reply:
'keys' is plural, and the implementation and documentation, and expectation for "KeysContainer"  is greatly simplified if we require a container of keys rather than "A container of keys, or just one key".

Reply:
I would recommend removing the "ensure_tuple" call, or at least not depend upon it.

Reply:
But other args will still support `ensure_tuple`, right?
For example, when I call `SimulateDelayd`:
```py
delay = SimulateDelayd(keys=("img",), delay_time=5)
```
@wyli @ericspod do you agree to use `ensure_tuple` for other args but can't use for `keys`?

Thanks.

Reply:
the intention of ensure_tuple is to simplify a bit of the syntax for example
``transform(keys=("img",), interp_order=("bilnear",))`` can also be written as 
``transform(keys="img", interp_order="bilinear")``.
my understanding is that this PR changes the behaviour, so it is probably fine